Makefile.frankfurt

```diff
@@ -221,10 +228,10 @@ dockerbuild: guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-APACHE_
 
 
 .PHONY: dockerrun
-dockerrun: guard-ENV_FILE guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-APACHE_PORT
+dockerrun: LoadEnvFile guard-ENV_FILE guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD guard-HTTP_PORT
```

Comment:
I think `LoadEnvFile` is not needed here, as `--env-file` is passed to `docker run` already.

Reply:
Right no LoadEnvFile here.

Reply:
the content of .env file has to be loaded before to substitute HTTP_PORT variable in the docker run command